CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.8.0 (2020-08-18)
```

Comment:
Probably need to update the changelog date whenever we release this, but otherwise LGTM.

Reply:
Went ahead and bumped this to the 19th.